fixed-requirements.txt

```diff
@@ -1,40 +1,40 @@
 # Packages versions fixed for the whole st2 stack
 # Note: greenlet is used by eventlet
 greenlet>=0.4.10,<0.5
-eventlet>=0.18.4,<0.19
+eventlet==0.21.0
 gunicorn==19.6.0
-kombu==3.0.37
+kombu==4.0.2
 # Note: amqp is used by kombu
-amqp==1.4.9
+amqp==2.1.4
 oslo.config>=1.12.1,<1.13
 oslo.utils<3.1.0
 six==1.10.0
 pyyaml>=3.12,<4.0
-requests[security]>=2.14.1,<2.15
+requests[security]>=2.18.1,<2.19
 apscheduler==3.3.1
-gitpython==2.1.3
-jsonschema>=2.5.0,<2.6
-mongoengine==0.11.0
+gitpython==2.1.5
+jsonschema==2.6.0
+mongoengine==0.13.0
```

Comment:
Also, as we just discussed offline, you are correct, this script shouldn't rely on other services to start first and with this change this is now indeed the case.

In the past we didn't do that, because registering internal trigger types talks to the database and we thought register-content will also be able to run on other servers without st2 and talk to st2 over API, but that's not the case and other functionality of the script also requires to be run from the server where st2 components are running.

And if we ever move it to the API, that's fine as well because then script won't require DB access  anymore.